.github/workflows/release.yaml

```diff
@@ -72,3 +73,40 @@ jobs:
             Freeciv21-${{github.ref_name}}.tar.gz.sha256
             Freeciv21-${{github.ref_name}}.zip
             Freeciv21-${{github.ref_name}}.zip.sha256
+
+  snapcraft:
+    name: "Snap Package"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install \
+            cmake \
+            ninja-build \
+            gettext \
+            qtbase5-dev \
+            libqt5svg5-dev \
+            libkf5archive-dev \
+            liblua5.3-dev \
+            libsqlite3-dev \
+            libsdl2-mixer-dev
+      - name: Configure
+        run: |
+          mkdir -p build/snap/local
+          cmake . -B build -G Ninja
+          cp data/icons/128x128/freeciv21-client.png build/snap/local
+      - uses: snapcore/action-build@v1
+        id: snapcraft
+        with:
+          path: build
+      - uses: snapcore/action-publish@v1
+        env:
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
+        with:
+          snap: ${{ steps.snapcraft.outputs.snap }}
+          release: beta
```

Comment:
How do you envision the move to a `stable` upload?

Reply:
Change it to stable and "edge" when we start up alphas again

Reply:
Should we document this procedure then?